libpixman.mk

```diff
@@ -0,0 +1,57 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS       += libpixman
+LIBPIXMAN_VERSION := 0.40.0
+DEB_LIBPIXMAN_V   ?= $(LIBPIXMAN_VERSION)
+
+libpixman-setup: setup
+	wget -q -nc -P $(BUILD_SOURCE) https://cairographics.org/releases/pixman-$(LIBPIXMAN_VERSION).tar.gz
+	$(call EXTRACT_TAR,pixman-$(LIBPIXMAN_VERSION).tar.gz,pixman-$(LIBPIXMAN_VERSION),libpixman)
+
+ifneq ($(wildcard $(BUILD_WORK)/libpixman/.build_complete),)
+libpixman:
+	@echo "Using previously built libpixman."
+else
+libpixman: libpixman-setup libpng16
```

Comment:
As far as I can tell libpng16 is not a build dependency.

Reply:
Just double-checked, and can confirm that is a build dependency.